projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/lang/zh_cn.json

```
@@ -816,7 +816,7 @@
 	"pneumaticcraft.gui.aphorismTileEditor": "格言瓷砖编辑器",
 	"pneumaticcraft.gui.aphorismTile.insert": "输入：",
 	"pneumaticcraft.gui.aphorismTile.insertItem": "物品……",
-	"pneumaticcraft.gui.aphorismTile.helpText": "§a§n格言瓷砖编辑器${br}${br}§e← → ↑ ↓：§r移动光标${br}§eHome：§r转到该行开始${br}§eEnd：§r转到该行结束${br}§e回车键：§r插入换行符${br}§e退格键：§r删除光标前的字符${br}§e删除键：§r删除光标后的字符${br}Alt + 删除键：§r删除整行${br}§eShift + 删除键：§r删除全部${br}§eEsc：§r结束编辑${br}§eAlt + §o字符§r：插入格式控制符${br}§7- 控制符：0-9，a-f，m，n，o，r",
+	"pneumaticcraft.gui.aphorismTile.helpText": "§a§n格言瓷砖编辑器${br}${br}§e← → ↑ ↓：§r移动光标${br}§eHome：§r转到该行开始${br}§eEnd：§r转到该行结束${br}§e回车键：§r插入换行符${br}§ebackspace：§r删除光标前的字符${br}§eDel：§r删除光标后的字符${br}Alt + Del：§r删除整行${br}§eShift + Del：§r删除全部${br}§eEsc：§r结束编辑${br}§eAlt + §o字符§r：插入格式控制符${br}§7- 控制符：0-9，a-f，m，n，o，r",
```

呃，这个要不要改.....
（至少Backspace可能要首字母大写）

---

我觉得两种都行，看你。

---

不过Shift没法译（
那就只有全部干成英文键名了
```suggestion
	"pneumaticcraft.gui.aphorismTile.helpText": "§a§n格言瓷砖编辑器${br}${br}§e← → ↑ ↓：§r移动光标${br}§eHome：§r转到该行开始${br}§eEnd：§r转到该行结束${br}§eEnter：§r插入换行符${br}§eBackspace：§r删除光标前的字符${br}§eDel：§r删除光标后的字符${br}Alt + Del：§r删除整行${br}§eShift + Del：§r删除全部${br}§eEsc：§r结束编辑${br}§eAlt + §o字符§r：插入格式控制符${br}§7- 控制符：0-9，a-f，m，n，o，r",
```